jsapp/js/components/common/icon.tsx

```diff
@@ -36,12 +38,13 @@ class Icon extends React.Component<IconProps, {}> {
     const size = this.props.size || DefaultSize;
     classNames.push(`k-icon--size-${size}`);
 
+    const color = this.props.color || DefaultColor;
+    classNames.push(`k-icon--color-${color}`);
+
     classNames.push('k-icon');
     classNames.push(`k-icon-${this.props.name}`);
 
-    return (
-      <i className={classNames.join(' ')}/>
-    );
+    return <i className={classNames.join(' ')} style={{color: `${color}`}} />;
```

Comment:
Continuing the above comment: instead of inline style, let's create and use class names :) We don't want any color, we want restricted list of colors